gilda/grounder.py

```diff
@@ -204,6 +219,47 @@ def _merge_equivalent_matches(scored_matches):
         # Return the list of unique entries
         return unique_entries
 
+    def get_models(self):
+        """Return a list of entity texts for which disambiguation models exist.
+
+        Returns
+        -------
+        list[str]
+            The list of entity texts for which a disambiguation model is
+            available.
+        """
+        return sorted(list(self.gilda_disambiguators.keys()))
+
+    def get_names(self, db, id, status=None, source=None):
+        """Return a list of entity texts corresponding to a given database ID.
+
+        Parameters
+        ----------
+        db : str
+            The database in which the ID is an entry, e.g., HGNC.
+        id : str
+            The ID of an entry in the database.
+        status : Optional[str]
+            If given, only entity texts with the given status e.g., "synonym"
+            are returned.
+        source : Optional[str]
+            If given, only entity texts from the given source e.g., "uniprot"
+            are returned.
+
+        Returns
+        -------
+        names: list[str]
+            A list of entity texts corresponding to the given database/ID
+        """
+        names = set()
```

Comment:
previous implementation maintained this as a list then cast to set at the end. I just switched to set from the beginning